packages/styles/scss/components/menu/_menu.scss

```diff
@@ -44,6 +44,7 @@
   }
 
   .#{$prefix}--menu--open {
+    inline-size: fit-content;
```

Comment:
Is this necessary still? I thought since the `${prefix}--menu--shown` was now guarded by `legacyAutoAlign` that this change wouldn't be needed.